.github/workflows/build_and_release.yml

```diff
@@ -137,7 +139,7 @@ jobs:
           case ${ID} in
```

Comment:
The `if` statement above needs to be set like it is here: https://github.com/performous/performous-docker/blob/master/build_performous.sh#L67-L71

(I tried to comment on the actual line, but you can't comment on lines that didn't change?)